### ISSUE TITLE
Not able to build with GCC 10

### ISSUE THREAD

**Issue by sdsddsd1** [opentyrian/opentyrian#2]
Hey,
since GCC 10 I cant build opentyrian anymore.
```
/tmp/opentyrian$ ➜  make
cc -std=iso9899:1999 -pedantic -MMD -Wall -Wextra -Wno-missing-field-initializers -O2 -Wl,-rpath,/usr/lib   -o opentyrian obj/animlib.o obj/arg_parse.o obj/backgrnd.o obj/config.o obj/config_file.o obj/destruct.o obj/editship.o obj/episodes.o obj/file.o obj/font.o obj/fonthand.o obj/game_menu.o obj/helptext.o obj/joystick.o obj/jukebox.o obj/keyboard.o obj/lds_play.o obj/loudness.o obj/lvllib.o obj/lvlmast.o obj/mainint.o obj/menus.o obj/mouse.o obj/mtrand.o obj/musmast.o obj/network.o obj/nortsong.o obj/nortvars.o obj/opentyr.o obj/opl.o obj/palette.o obj/params.o obj/pcxload.o obj/pcxmast.o obj/picload.o obj/player.o obj/scroller.o obj/setup.o obj/shots.o obj/sizebuf.o obj/sndmast.o obj/sprite.o obj/starlib.o obj/std_support.o obj/tyrian2.o obj/varz.o obj/vga256d.o obj/vga_palette.o obj/video.o obj/video_scale.o obj/video_scale_hqNx.o obj/xmas.o -lm -lSDL -lpthread  -lSDL_net
/usr/bin/ld: obj/destruct.o:(.bss+0x0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0xc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x10): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/destruct.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0xc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x10): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/game_menu.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0xc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x10): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/jukebox.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x4): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x8): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0xc): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x10): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x20): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/lds_play.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0xe8): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0xec): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0xf0): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0xf4): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x100): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x120): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x220): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x224): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x228): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x240): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/loudness.o:(.bss+0x1050): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x14): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x18): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x1c): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x20): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x30): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x60): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x160): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x164): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x168): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0x180): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/mainint.o:(.bss+0xf90): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x1c0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x1c4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x1c8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x1cc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x1d0): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x200): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x300): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x304): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x308): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x320): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/nortsong.o:(.bss+0x1130): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0xc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x10): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/opentyr.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x10): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0xc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0x0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/opl.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x4): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x8): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/params.o:(.bss+0xc): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x10): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x20): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/params.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/params.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0xc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x10): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x40): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x140): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x144): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x148): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0x160): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/scroller.o:(.bss+0xf70): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7638): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x763c): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7640): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7644): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7650): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7680): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7780): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7784): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x7788): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x77a0): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/tyrian2.o:(.bss+0x85b0): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53ab0): multiple definition of `tremtab_add'; obj/config.o:(.bss+0x1494): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53ab4): multiple definition of `tremtab_pos'; obj/config.o:(.bss+0x1498): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53ab8): multiple definition of `vibtab_add'; obj/config.o:(.bss+0x149c): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53abc): multiple definition of `vibtab_pos'; obj/config.o:(.bss+0x14a0): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53ac0): multiple definition of `wave_sel'; obj/config.o:(.bss+0x14b0): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53ae0): multiple definition of `adlibreg'; obj/config.o:(.bss+0x14e0): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53be0): multiple definition of `opl_index'; obj/config.o:(.bss+0x15e0): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53be4): multiple definition of `status'; obj/config.o:(.bss+0x15e4): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53be8): multiple definition of `int_samplerate'; obj/config.o:(.bss+0x15e8): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x53c00): multiple definition of `op'; obj/config.o:(.bss+0x1600): first defined here
/usr/bin/ld: obj/varz.o:(.bss+0x54a10): multiple definition of `chip_num'; obj/config.o:(.bss+0x2410): first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:128: opentyrian] Error 1
```

**Comment by carlreinke** [comment]
From https://gcc.gnu.org/gcc-10/changes.html
> GCC now defaults to -fno-common. As a result, global variable accesses are more efficient on various targets. In C, global variables with multiple tentative definitions now result in linker errors. With -fcommon such definitions are silently merged during linking.

I'll implement a real fix, but to get it working for now, do
```
make clean
make CFLAGS=-fcommon
```

**Comment by sdsddsd1** [comment]
I tried to export the CFLAGS before but without success. Thanks for the tip!

**Comment by carlreinke** [comment]
Fixed by 962ee8fc46ca51691bde1c8c1022dacbe8a037ed.
